notebooks/263-stable-diffusion-torchdynamo-backend/263-stable-diffusion-torchdynamo-backend.ipynb

```diff
@@ -0,0 +1,420 @@
+{
```

Comment:
<p>having 2 pipelines in memory in the same time can requires extra RAM. Maybe we can split demonstration that allow users who have less RAM successfully try notebook (free memory after first pipeline usage, then run the second) and also allow us highlight that the same actions required for both tasks.</p><p> </p><p>When we run ov directly we combain both pipelines in one and reuse memory running both task, but with torch.compile it is not applicable</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>